ci/jenkins/jobs/macros.yaml

```diff
@@ -12,145 +12,7 @@
-          sed -i "s|SSHAUTHORIZEDKEYS|${publickey}|g" "${WORKSPACE}/jenkins/out/cluster.yaml"
-          sed -i "s/CLUSTERNAMESPACE/${cluster}/g" "${WORKSPACE}/jenkins/out/namespace.yaml"
-
-          echo "=== network spec value substitution==="
-          cluster_defaults="${WORK_HOME}/utils/CLUSTERDEFAULTS"
-          while IFS= read -r line
-          do
-            IFS='=' read -ra kv <<< "$line"
-            sed -i "s|${kv[0]}|${kv[1]}|g" "${WORKSPACE}/jenkins/out/cluster.yaml"
-          done < "$cluster_defaults"
-
-          echo '=== Create a cluster in management cluster ==='
-          kubectl apply -f "${WORKSPACE}/jenkins/out/namespace.yaml"
-          kubectl apply -f "${WORKSPACE}/jenkins/out/cluster.yaml"
-
-          echo '=== Wait for workload cluster secret for 10 min ==='
-          for t in {1..10}
-          do
-            sleep 1m
-            echo '=== Get kubeconfig (try for 1m) ==='
-            if kubectl get secret/${cluster}-kubeconfig -n${cluster} ; then
-              kubectl get secret/${cluster}-kubeconfig -n${cluster} -o json \
-                | jq -r .data.value \
-                | base64 --decode \
-                > "${WORKSPACE}/jenkins/out/kubeconfig"
-              touch jenkins/SECRET_EXIST
-              break
-            fi
-          done
-
-          if !(test -f jenkins/SECRET_EXIST); then
-            echo "=== Failed to get secret ==="
-            saveLogs
-            kubectl delete ns ${cluster}
-            exit 1
-          else
-            export KUBECONFIG="${WORKSPACE}/jenkins/out/kubeconfig"
-            echo "=== Waiting all nodes up for 10 min ==="
-
-            set +e
-            for t in {1..10}
-            do
-              sleep 1m
-              echo "=== Get node (try for 1m) ==="
-              mdNum="$(kubectl get node | grep -c ${cluster}-md)"
-              if [ "${mdNum}" == "2" ]; then
-                echo "=== Setup workload cluster succeeded ==="
-                exit 0
-              fi
-            done
-            set -e
-
-            echo "=== Failed to make all nodes up ==="
-            saveLogs
-            KUBECONFIG="${WORK_HOME}/.kube/config" kubectl delete ns ${cluster}
-            exit 1
-          fi
-
-- builder:
-    name: builder-workload-cluster-cleanup
-    builders:
-      - shell: |-
-          #!/bin/bash
-          set -ex
-          echo '=== Clean up workload cluster ==='
-          export KUBECONFIG="/var/lib/jenkins/.kube/config"
-
-          cluster="${JOB_NAME}-${BUILD_NUMBER}"
-          kubectl delete ns ${cluster}
-          rm -rf "${WORKSPACE}/jenkins"
-
-          echo "=== Cleanup workload cluster ${cluster} succeeded ==="
-
-          if !(test -f TEST_FAILURE); then
-            echo "=== SUCCESS !!! ==="
-            exit 0
-          fi
-          echo "=== FAILURE !!! ==="
-          exit 1
+            ./ci/test-vmc.sh --testcase integration
```

Comment:
what will be the cluster name in this case?

Reply:
Integration test doesn't need one because it runs on a fixed VM.

Reply:
Oh that's right there is no K8s cluster for integration tests. But I though we created a new VM every time? Is it the case or do we use the same VM always?

Reply:
The same VM gets reverted to a snapshot everytime.

Reply:
that's right, thanks for the reminder